src/QuickMarcEditor/utils.js

```diff
@@ -546,6 +546,12 @@ const validateMarcHoldingsRecord = (marcRecords, locations) => {
     return <FormattedMessage id="ui-quick-marc.record.error.location.multiple" />;
   }
 
+  if (locationRecords.length) {
```

Comment:
I think we don't need to check `if (locationRecords.length)` because there is an `if` before that checks this case